src/frontend/src/lib/types/route.ts

```diff
@@ -0,0 +1,3 @@
+export type Route = 'transactions' | 'tokens' | 'settings' | 'explore';
```

Comment:
I'm not convinced by have a global type route that actually does not match the routing of `$page`.

`let route: ...` was just a handy local variable.

In NNS dapp if I remember correctly, we can have a look, I think we use an enum that contains the route names.

Reply:
For example in NNS dapp it's

```
export enum AppPath {
  Accounts = "/accounts",
  Wallet = "/wallet",
```

With `/accounts` being the href.

```
export const ROUTE_ID_GROUP_APP = "/(app)";
export const ROUTE_ID_GROUP_UNIVERSE = "/(u)";
```


```
$: routes = [
    {
      context: "accounts",
      href: AppPath.Tokens,
      selected: isSelectedPath({
        currentPath: $pageStore.path,
        paths: [AppPath.Accounts, AppPath.Wallet, AppPath.Tokens],
      }),
      title: $i18n.navigation.tokens,
      icon: IconWallet,
    },
    ...
```